src/olympia/activity/tests/test_utils.py

```diff
@@ -516,7 +516,7 @@ def test_send_activity_mail():
     assert len(mail.outbox) == 1
     assert mail.outbox[0].body == message
     assert mail.outbox[0].subject == subject
-    reference_header = '{addon}/{version}@{site}'.format(
+    reference_header = '<{addon}/{version}@{site}>'.format(
```

Comment:
This is similar to `Christopher Grebs <cg@webshox.org>` wrapping, doesn't and should be escaped.